### ISSUE TITLE
Question: Can I contribute?

### ISSUE THREAD

**Issue by yarsiemanym** [stpain/guildbook#56]
Hey, guys.  The officers in my guild have been taking Guildbook for a spin.  It seems like a great idea and we'd love to use it but it also looks like it still needs some work.  I've been professional developer for 15 years and wouldn't mind contributing to this project where ever I can. If there are any simple bugs you want to assign to me I could take a stab at them.  I'd submit all changes as a PR for your approval.

**Comment by stpain** [comment]
Hi, your timing is good, i have a disc issue in my back so any dev work is minimal at the moment.

If you have anything you know needs a fix feel free to go ahead, are you familiar with git and curseforge, i could add you to the project. i must admit im very much a beginner and the code isnt great.

**Comment by yarsiemanym** [comment]
I'm very familiar with Git and have used Curseforge pretty extensively but never developed any addons before.  I'll clone the repo and poke around a bit.  Get familiar with the current code and see where I can help.

Hope your back feels better soon.

**Comment by stpain** [comment]
Sounds good, maybe you can help me learn more about git!

I just merged some translations fixes and will push with a tag to send the update to curse in a moment. Nothing major changed with the code though but in case you want a fresh pull etc.

Have you used github desktop? I tried to do the merge/commit through that but got confused.

**Comment by yarsiemanym** [comment]
Definitely!  Git can be a bit tricky.  Especially when you get multiple people working on the same code and have to resolve conflicts and whatnot.  I _have_ used Git Desktop but I usually use the command line
